File: 240924/함수를 이용한 369 게임/369-games-using-functions.py
a, b = map(int, input().split())

def isThree(n):
    if n % 3 == 0:
        return True
    return False

def isNum(n):
    while n > 0:
        if n % 10 == 3 or n % 10 == 6 or n % 10 == 9:
            return True
        n //= 10
    return False

result = 0
for i in range(a, b+1):
    if isNum(i) or isThree(i):
        result += 1

print(result)

File: 240924/함수를 이용한 369 게임/369-games-using-functions.c
#include <stdio.h>

int isNum(int n) {
    while(n > 0) {
        if (n % 10 == 3 || n % 10 == 6 || n % 10 == 9) return 1;
        n /= 10;
    }
    return 0;
}

int isThree(int n) {
    if (n % 3 == 0) return 1;
    return 0;
}

int main() {
    int a, b, i;
    scanf("%d %d", &a, &b);
    int result = 0;
    for (i=a; i<=b; i++) {
        if (isNum(i) == 1 || isThree(i) == 1) {
            result += 1;
        } 
    }
    printf("%d", result);
    return 0;
}